benchmark/single-source/ByteSwap.swift

```diff
@@ -18,15 +18,25 @@ import TestsUtils
 
 // a naive O(n) implementation of byteswap.
 func byteswap_n(_ a: UInt64) -> UInt64 {
-  var result = ((a & 0x00000000000000FF) << 56)
-  result |=    ((a & 0x000000000000FF00) << 40)
-  result |=    ((a & 0x0000000000FF0000) << 24)
-  result |=    ((a & 0x00000000FF000000) <<  8)
-  result |=    ((a & 0x000000FF00000000) >>  8)
-  result |=    ((a & 0x0000FF0000000000) >> 24)
-  result |=    ((a & 0x00FF000000000000) >> 40)
-  result |=    ((a & 0xFF00000000000000) >> 56)
-  return result
+#if swift(>=4)
```

Comment:
Do you mean you want the compatibility mode to take the old path or the new? It will take the old.